src/Lykke.Service.Regulation.AzureRepositories/ClientRegulationRepository.cs

```diff
@@ -14,27 +18,126 @@ public ClientRegulationRepository(INoSQLTableStorage<ClientRegulationEntity> tab
             _tableStorage = tableStorage;
         }
 
-        public async Task<IClientRegulation> GetAsync(string clientId)
+        public async Task<IClientRegulation> GetAsync(string clientId, string regulationId)
         {
             var partitionKey = ClientRegulationEntity.GeneratePartitionKey();
-            var rowKey = ClientRegulationEntity.GenerateRowKey(clientId);
+            var rowKey = ClientRegulationEntity.GenerateRowKey(clientId, regulationId);
```

Comment:
the first parameter of GenerateRowKey, is regulationId and the second is clientId, but called vise versa